README.md

```diff
@@ -79,8 +79,8 @@ You can find our current feature and bug priorities for forthcoming
 releases in the [projects section](https://github.com/ethereum/solidity/projects).
 
 ## Maintainers
-* [@axic](https://github.com/axic)
-* [@chriseth](https://github.com/chriseth)
+The Solidity programming language and the solc compiler are open-source, community projects governed by 
```

Comment:
I think we usually refer to it as just "Solidity compiler"? There's one instance of it even in this file.
```suggestion
The Solidity programming language and compiler are open-source, community projects governed by 
```

Reply:
jup good point.